openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```diff
@@ -5,6 +5,11 @@
     "label": "Maya",
     "is_file": true,
     "children": [
+        {
+            "type": "boolean",
+            "key": "open_workfile_post_initialization",
+            "label": "Open Workfile Post Initialization"
+        },
```

Comment:
Should this maybe have a label in the documentation describing when you might want to use this?

Reply:
Feel like this is generic enough to be use for other use cases, so I would not mention Xgen here specifically. But we should elaborate on what the setting does. Good point!

Reply:
I meant more like - what's the reason the feature would not be enabled by default? Is there any? etc.

Reply:
Consulted with @antirotor about this and we cant think of a reason why it would be bad to enable this by default. We are just scared to change just a global setting...